agent/agent.go

```diff
@@ -114,8 +114,8 @@ type agent struct {
 	sshServer     *ssh.Server
 
 	enableWireguard      bool
-	wg                   *peerwg.WireguardNetwork
-	postKeys             PostKeys
+	network              *peerwg.Network
+	postKeys             UploadWireguardKeys
```

Comment:
```suggestion
	uploadWireguardKeys             UploadWireguardKeys
```
formatting shit but u get the point